src/permutat.cc

```diff
@@ -1125,6 +1125,40 @@ UInt LargestMovedPointPerm(Obj perm)
         return LargestMovedPointPerm_<UInt4>(perm);
 }
 
+/****************************************************************************
+**
+*F  SmallestMovedPointPerm( <perm> ) smallest point moved by perm
+**
+**  'SmallestMovedPointPerm' returns  the  smallest  positive  integer that is
+**  moved by the permutation <perm>.
+*/
+
+// Import 'Infinity', as return value for identity permutation
```

Comment:
as the return value for the/an identity pmerutation/permutations (not sure how one would say it in english)